source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -4045,12 +4074,12 @@ void Graph::drawGraph(ImVec2 mousePos)
         // Add new link
         if (ed::BeginCreate())
         {
-            ed::PinId inputPinId, outputPinId, filterPinId;
```

Comment:
Also tweaked the terminology here. imgui's node editor simply returns a start/end -- could be output to input or input to output. Changed the terminology here to reflect that.